bluebottle/funding/tests/test_admin.py

```diff
@@ -153,6 +154,18 @@ def test_donation_reward(self):
         self.assertTrue(second.title in response.content.decode('utf-8'))
         self.assertFalse(third.title in response.content.decode('utf-8'))
 
+    def test_contribution_date(self):
+        donation = DonorFactory.create(
+            activity=self.funding,
+            amount=Money(100, 'EUR')
+        )
+        donation.states.succeed(save=True)
+        self.client.force_login(self.superuser)
+        response = self.client.get(self.admin_url)
```

Comment:
`self.admin_url` refers to a list of Funding activities. You are trying to prove that the first column of a Donor list holds a value, right? So you should use `reverse('funding_donor_changelist')`. Best move this to another TestCase.